website/src/layouts/marketing.js

```diff
@@ -68,13 +68,26 @@ const FooterConstraint = styled.div`
 `;
 
 const LayoutMarketing = props => {
+  // State for the top menu
+  const [isTopMenuOpen, setIsTopMenuOpen] = React.useState(false);
+  const toggleTopMenu = React.useCallback(() => {
+    setIsTopMenuOpen(prev => !prev);
+  }, [setIsTopMenuOpen]);
+  const closeTopMenu = React.useCallback(() => {
+    setIsTopMenuOpen(false);
+  }, [setIsTopMenuOpen]);
```

Comment:
It somehow still amazes me that we always write 7 lines for a boolean toggle 😛.

Reply:
Yeah, especially when you have more boolean toggles together 😅